core/src/com/unciv/models/ruleset/IHasUniques.kt

```diff
@@ -1,14 +1,17 @@
 package com.unciv.models.ruleset
 
 import com.unciv.models.ruleset.unique.Unique
+import com.unciv.models.ruleset.unique.UniqueTarget
 import com.unciv.models.ruleset.unique.UniqueType
 
 /**
  * Common interface for all 'ruleset objects' that have Uniques, like BaseUnit, Nation, etc.
  */
 interface IHasUniques {
     var uniques: ArrayList<String> // Can not be a hashset as that would remove doubles
+    // I bet there's a way of initializing these without having to override it everywhere...
```

Comment:
Never found it. Interfaces can have default implementations for methods, but not for properties with backing fields.